chart/kubeapps/crds/apprepository-crd.yaml

```diff
@@ -10,4 +10,9 @@ spec:
     plural: apprepositories
     shortNames:
       - apprepos
+  {{- if semverCompare "<1.19-0" (include "common.capabilities.kubeVersion" .) -}}
```

Comment:
I guess here we are solving this: https://github.com/kubeapps/kubeapps/issues/2169#issuecomment-728910010
No?

Reply:
I guess, but it'd be nice if you give that a try

Reply:
Reverting these changes, since only pure YAML (no templates) is accepted for CRDs